infrastructure/README.md

```diff
@@ -4,28 +4,29 @@
 
 1. Add new module to `pipeline.tf`, named with the date.
 2. From `./infrastructure`, plan and apply Terraform (`terraform init` might be required first)
+
+At this stage you could chose to do it manually (see instructions below), or run `./create_pipeline.sh`. If you chose to do so, make sure you run it with its required arguments; `./create_pipeline.sh PIPELINE_DATE=[NEW PIPELINE DATE] AWS_PROFILE=catalogue-developer BUILDKITE_COMMIT=dev`.
```

Comment:
I would avoid having both a set of instructions and a script to avoid one diverging without the other. If there's a script that works, we can point at that and add comments inline that document what it's doing.

Reply:
Thank you for all your comments, I've made all the changes now!